src/BridgedRoom.ts

```diff
@@ -429,6 +450,10 @@ export class BridgedRoom {
         if (message.user_id === this.team!.user_id) {
             return;
         }
+        if (this.recentSlackMessages.includes(`reactadd:${message.reaction}:${message.user_id}`)) {
+            // We sent this, ignore.
```

Comment:
We send this, add it to the events db, and I think you need the parent event ts for that, even if you don't need it to make the reaction unique.

Reply:
Leaving the DB stuff to you :stuck_out_tongue: